packages/server/src/server.ts

```diff
@@ -166,22 +175,17 @@ export class ProsopoServer {
      * @param maxVerifiedTime
      */
     public async verifyContract(user: string, maxVerifiedTime = DEFAULT_MAX_VERIFIED_TIME_CONTRACT) {
-        try {
-            const contractApi = await this.getContractApi()
-            this.logger.info('Provider URL not provided. Verifying with contract.')
-            const correctCaptchaBlockNumber = (await contractApi.query.dappOperatorLastCorrectCaptcha(user)).value
-                .unwrap()
-                .unwrap()
-                .before.valueOf()
-            const verifyRecency = await this.verifyRecency(correctCaptchaBlockNumber, maxVerifiedTime)
-            const isHuman = (await contractApi.query.dappOperatorIsHumanUser(user, this.config.solutionThreshold)).value
-                .unwrap()
-                .unwrap()
-            return isHuman && verifyRecency
-        } catch (error) {
-            // if a user is not in the contract it errors, suppress this error and return false
-            return false
-        }
+        const contractApi = await this.getContractApi()
```

Comment:
The previous comment says:

// if a user is not in the contract it errors, suppress this error and return false

When did this change?

Reply:
I am not sure
Should I change this?

Reply:
I have done

Reply:
Please don't change this. The error is suppressed for a reason.